neo/Network/P2P/Payloads/GetFullBlocksPayload.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class GetFullBlocksPayload : ISerializable
+    {
+        private const ushort MaxBlocksCount = 500;
+        public uint IndexStart;
+        public ushort Count;
+
+        public int Size => sizeof(uint) + sizeof(ushort);
+
+        public static GetFullBlocksPayload Create(uint index_start, ushort count = 500)
+        {
+            return new GetFullBlocksPayload
+            {
+                IndexStart = index_start,
+                Count = count
+            };
+        }
+
+        void ISerializable.Deserialize(BinaryReader reader)
+        {
+            IndexStart = reader.ReadUInt32();
+            Count = reader.ReadUInt16();
+            if (Count == 0 || Count > MaxBlocksCount) throw new FormatException();
```

Comment:
@shargon, maybe we can use the same logic here: https://github.com/neo-project/neo/blob/0a7cba7280418a96b213ffe46ddcb738b85409b9/neo/Network/P2P/Payloads/InvPayload.cs#L37-L43

In view of this recent commit https://github.com/neo-project/neo/commit/c9e853243c19632185dd60d08a9fe3cb04bf32c3

Reply:
@vncoelho I'm not sure I follow you, are you suggesting this?
```C#
if (Count == 0) throw new FormatException();
if (Count > MaxBlocksCount) 
    Count = MaxBlocksCount
```
If not, please be explicit in your suggestion. Thanks.

Reply:
I mean, @ixje, that last commit I mentioned (that you authored) verified the limit  on an upper layer, right?
I suggest that we revert that commit and make it like this, on the `Deserialization process`.

What do you think, @shargon?

Reply:
@vncoelho I think it's a good observation that verification happens in different layers depending on the payload. In the commit/PR you linked I only fixed the incomplete verification. 

I think the overall related code could be more consistent by doing all verification in the same layer. I'd suggest to create a separate issue for that as it is a bit off topic for this PR. I want to avoid people thinking there's still work todo here by having an unresolved conversation (so I'll resolve it for just that matter). Thanks !

Reply:
Yes, that should be a different issue. I just mentioned here because I saw the code and the idea come into the mind.

Makes sense, @ixje. I will resolve this one.

Reply:
@ixje, take a look at this comment https://github.com/neo-project/neo/pull/1194#issuecomment-550373037

Perhaps we can revert that PR.